### PR TITLE
feat: Center datadog disclaimer in left sidebar menu

### PR DESCRIPTION
**Pull request by przemvs**
Waiting for: https://github.com/wireapp/wire-web-packages/pull/6467

## Description

Centered datadog disclaimer icon and text. Also changed icon info to warning icon.

## Screenshots/Screencast (for UI changes)

Before:
image

After:
image

## Checklist

- [ ] PR has been self reviewed by the author;
- [ ] Hard-to-understand areas of the code have been commented;
- [ ] If it is a core feature, unit tests have been added;

### Important details for the reviewers

(Delete this section if unnecessary)

- use (x) data
- can be reviewed commit-by-commit
- be sure to look at ...

### CODE REVIEW

File: src/script/page/LeftSidebar/panels/Conversations/ConversationTabs/ConversationTabs.styles.ts [hunk 1]
```diff
@@ -23,10 +23,17 @@ export const footerDisclaimer: CSSObject = {
   display: 'grid',
   gap: '10px',
   gridTemplateColumns: '1fr auto',
+  marginBottom: '4px',
```
Comment (e-maad) [inline src/script/page/LeftSidebar/panels/Conversations/ConversationTabs/ConversationTabs.styles.ts:26]: ~~The spacing between icons are not consistent.
Is it intentional?~~

Edit: Fixed after package update.

Screenshot 2024-08-22 at 10 16 05

Screenshot 2024-08-22 at 10 15 52

File: src/script/page/LeftSidebar/panels/Conversations/ConversationTabs/ConversationTabs.styles.ts [hunk 2]
```diff
@@ -23,10 +23,17 @@ export const footerDisclaimer: CSSObject = {
...
   padding: '8px',
+
+  '.footer-disclaimer-tooltip': {
+    alignItems: 'center',
+    display: 'flex',
+  },
```
Comment (e-maad) [inline src/script/page/LeftSidebar/panels/Conversations/ConversationTabs/ConversationTabs.styles.ts]: Is there any specific reason to add these style under the class, not as CSS object?

**Review by e-maad — COMMENTED** [review]
Looks good 🚀 
Added few concerns, please have a look.

File: src/script/page/LeftSidebar/panels/Conversations/ConversationTabs/ConversationTabs.styles.ts (hunk 2 above)
Comment (przemvs) [reply src/script/page/LeftSidebar/panels/Conversations/ConversationTabs/ConversationTabs.styles.ts]: Updated ;)

**Review by e-maad — APPROVED** [review]
Nicely done 🚀

**Pull request merged**